src/app/app-routing.module.ts

```diff
@@ -0,0 +1,24 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { HomeComponent } from './home/home.component';
+import { ActivateUserComponent } from './activate/activate-user/activate-user.component';
+import { DeletedUserComponent } from './deleted/deleted-user/deleted-user.component';
+
+const routes: Routes = [
+  { path: "", component: HomeComponent },
+   { path: "activate", component: ActivateUserComponent},
```

Comment:
TO have better control over the route, create an enum to store all the route strings there.

This is okay for now. Try to implement the same in upcoming changes